vendor/glfw/bindings/bindings.odin

```diff
@@ -38,7 +38,13 @@ when ODIN_OS == .Windows {
 		}
 	}
 } else {
-	foreign import glfw "system:glfw"
+	when GLFW_SHARED {
+		foreign import glfw "system:glfw"
+	} else {
+		foreign import glfw { 
```

Comment:
Would be good to add a `when !#exists("path") { #panic("no static library found, add it at ....") }` here.

See https://github.com/odin-lang/Odin/blob/daa3bb9a3beb8a4551ef4c8bdeefcaaaf11ed30d/vendor/stb/image/stb_image_resize.odin#L15 for an example.

Reply:
Done